docs/sources/flow/getting-started/migrating-from-operator.md

```diff
@@ -0,0 +1,262 @@
+
+This guide will provide some steps to get started with Grafana Agent for users coming from Grafana Agent Operator.
+
+## Deploy Grafana Agent with Helm
+
+1. You will need to create a `values.yaml` file, which contains options for how to deploy your agent. You may start with the [default values](https://github.com/grafana/agent/blob/main/operations/helm/charts/grafana-agent/values.yaml) and customize as you see fit, or start with this snippet, which should be a good starting point for what the operator does:
+
+    ```yaml
+    agent:
+      mode: 'flow'
+      configMap:
+        create: true
+      clustering:
+        enabled: true
+    controller:
+      type: 'statefulset'
+    replicas: 2
+    crds:
+      create: false
+    ```
+
+    This config will deploy Grafana Agent as a `StatefulSet` using the built in [clustering](https://grafana.com/docs/agent/latest/flow/concepts/clustering/) functionality to allow distributing scrapes across all Agent Pods. 
+    
+    This is not the only deployment mode possible. For example, you may desire a `DaemonSet` if collecting host level logs or metrics. See [this page](https://grafana.com/docs/agent/latest/flow/setup/deploy-agent/) for more details about different topologies.
+
+2. Create a flow config file, `agent.river`.
+
+    You can add any config you need directly to this file.
+
+3. Install the grafana helm repository:
+
+    ```
+    helm repo add grafana https://grafana.github.io/helm-charts
+    helm repo update
+    ```
+
+4. Create a helm relase. You may name the relase anything you like. Here we are installing a release named `grafana-agent-metrics` in the `monitoring` namespace.
+
+    ```
+    helm upgrade grafana-agent-metrics grafana/grafana-agent -i -n monitoring -f values.yaml --set-file agent.configMap.content=agent.river
+    ```
+
+    This command uses the `--set-file` flag to pass the config file as a helm value, so that we can continue to edit it as a regular river file.
+
+## Convert `MetricsIntances` to flow components
+
+A `MetricsInstance` resource primarily defines:
+
+- The remote endpoint(s) Grafana Agent should send metrics to.
+- Which `PodMonitor`, `ServiceMonitor`, and `Probe` resources this Agent should discover.
+
+These functions can be done in Grafana Agent Flow with the `prometheus.remote_write`, `prometheus.operator.podmonitors`, `prometheus.operator.servicemonitors`, and `prometheus.operator.probes` components respectively.
+
+This is a river sample that is equivalent to the `MetricsInstance` from our [operator guide](https://grafana.com/docs/agent/latest/operator/deploy-agent-operator-resources/#deploy-a-metricsinstance-resource):
+
+```river
+
+// read the credentials secret for remote_write authorization
+remote.kubernetes.secret "credentials" {
+  namespace = "monitoring"
+  name = "primary-credentials-metrics"
+}
+
+prometheus.remote_write "primary" {
+    endpoint {
+        url = "https://PROMETHEUS_URL/api/prom/push"
```

Comment:
I think that `api/prom/push` is an older [push endpoint for loki](https://grafana.com/docs/loki/latest/reference/api/#examples-11), mimir [expects api/v1/push](https://grafana.com/docs/mimir/latest/references/http-api/), and prometheus expects [api/v1/write](https://prometheus.io/docs/prometheus/latest/storage/#overview).

Maybe we should use the mimir one or stub the whole thing out and move the possibilities to the note below the snippet?